x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.test.tsx

```diff
@@ -200,6 +204,272 @@ describe('alerts default_config', () => {
     });
   });
 
+  describe('getAlertsDefaultModel', () => {
+    test('returns correct model for Basic license', () => {
+      const licenseServiceMock = createLicenseServiceMock();
+      licenseServiceMock.isPlatinumPlus.mockReturnValue(false);
+      const model = getAlertsDefaultModel(licenseServiceMock);
+
+      const expected = {
+        ...tableDefaults,
+        showCheckboxes: true,
+        columns: [
+          { columnHeaderType: 'not-filtered', id: '@timestamp', initialWidth: 200 },
+          {
+            columnHeaderType: 'not-filtered',
+            displayAsText: 'Rule',
+            id: 'kibana.alert.rule.name',
+            initialWidth: 180,
+            linkField: 'kibana.alert.rule.uuid',
+          },
+          {
+            columnHeaderType: 'not-filtered',
+            displayAsText: 'Assignees',
+            id: 'kibana.alert.workflow_assignee_ids',
+            initialWidth: 190,
+          },
+          {
+            columnHeaderType: 'not-filtered',
+            displayAsText: 'Severity',
+            id: 'kibana.alert.severity',
+            initialWidth: 105,
+          },
+          {
+            columnHeaderType: 'not-filtered',
+            displayAsText: 'Risk Score',
+            id: 'kibana.alert.risk_score',
+            initialWidth: 100,
+          },
+          {
+            columnHeaderType: 'not-filtered',
+            displayAsText: 'Reason',
+            id: 'kibana.alert.reason',
+            initialWidth: 450,
+          },
+          { columnHeaderType: 'not-filtered', id: 'host.name' },
+          { columnHeaderType: 'not-filtered', id: 'user.name' },
+          { columnHeaderType: 'not-filtered', id: 'process.name' },
+          { columnHeaderType: 'not-filtered', id: 'file.name' },
+          { columnHeaderType: 'not-filtered', id: 'source.ip' },
+          { columnHeaderType: 'not-filtered', id: 'destination.ip' },
+        ],
+      };
+      expect(model).toEqual(expected);
+    });
+
+    test('returns correct model for Plutinum license', () => {
```

Comment:
just a minor spelling item for this one and the other one below. `plutinum` => `platinum` 😄  Thanks for adding these tests!

Reply:
thanks for catching this